ls1intum/Artemis
# [Docs] file upload exercises and second correction in exam mode

### Motivation and Context


I added missing documentation for the second correction of exams, and documentation for file-upload exercises.

### Steps for Testing

easiest way to "test" is, to check out the repository locally. Then build the docs. See the README in the docs folder.
